sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/debug/StackTraceSourceInfo.java

```diff
@@ -41,8 +41,9 @@ public String multiLineDebugString() {
     return "\tat unknown source";
   }
 
-  private static boolean isOTelSdkStackTrace(StackTraceElement e) {
+  private static boolean isUninterestingStackTrace(StackTraceElement e) {
     return (e.getClassName() != null)
-        && e.getClassName().startsWith("io.opentelemetry.sdk.metrics");
+        && (e.getClassName().startsWith("io.opentelemetry.sdk.metrics")
+            || e.getClassName().startsWith("java.lang"));
```

Comment:
what's the case for this one?

Reply:
Thread.currentThread().getStackTrace() includes the Thread class's stack.